flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/DefaultJobManagerRunnerRegistryTest.java

```diff
@@ -164,7 +164,8 @@ void testFailingLocalCleanup() {
                 .hasExactlyElementsOfTypes(ExecutionException.class, expectedException.getClass())
                 .last()
                 .isEqualTo(expectedException);
-        assertThat(testInstance.isRegistered(jobManagerRunner.getJobID())).isFalse();
+        // Since the cleanup failed, the JobManagerRunner is expected to not have been unregistered.
```

Comment:
nit: you could use the `.as()` method of JUnit5 to add a assert message rather than a comment. It would serve the same function as a comment but would also expose the description programmatically.

Reply:
Done, thanks!